DEV_2/DEV_2UnitTest/StringLettersToSoundReplacerTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace DEV_2UnitTest
+{
+    using DEV_2;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    /// <summary>
+    /// <see cref="StringLettersToSoundReplacer"/> methods test.
+    /// </summary>
+    [TestClass]
+    public class StringLettersToSoundReplacerTest
+    {
+        /// <summary>
+        /// The replacer.
+        /// </summary>
+        private StringLettersToSoundReplacer replacer;
+
+        /// <summary>
+        /// The enter word test.
+        /// </summary>
+        /// <param name="word">
+        /// The word.
+        /// </param>
+        /// <param name="expectedResult">
+        /// The expected result.
+        /// </param>
+        [DataRow("злее", "злэйэ")]
+        [DataRow("ею", "йэйу")]
+        [DataRow("БОЛЬШИЕ", "большийэ")]
+        [DataTestMethod]
+        public void ProcessingVowelsIntoSoundsTest(string word, string expectedResult)
+        {
+            this.replacer = new StringLettersToSoundReplacer(word);
+            Assert.AreEqual(expectedResult, this.replacer.ProcessingVowelsIntoSounds());
```

Comment:
it is better  to create a variable for actual result "this.replacer.ProcessingVowelsIntoSounds()"